.github/dependabot.yml

```
@@ -1,12 +1,18 @@
-# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+# https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference
 
 version: 2
 updates:
   - package-ecosystem: "maven"
     directory: "/"
     schedule:
-      interval: "monthly"
+      interval: "weekly"
+      day: "friday"
+    ignore:
+      # BOM and build dependencies are excluded, they are released far too often and matter too little
+      - dependency-name: "io.jenkins.tools.bom:bom-2.*.x"
```

```suggestion
      # Build dependencies are excluded, they are released far too often and matter too little
```
BOM being removed by https://github.com/jenkinsci/strict-crumb-issuer-plugin/pull/22

---

Reverted.